nc-news-fe/api.js

```diff
@@ -1,11 +1,31 @@
 import axios from 'axios'
 
-const getAllArticles = () => {
-    return axios.get("https://nc-news-a8e3.onrender.com/api/articles")
+const api = axios.create({
+    baseURL: "https://nc-news-a8e3.onrender.com/api"})
+
+export const getAllArticles = () => {
+    return api.get("/articles")
         .then(res => res.data.articles)
         .catch(error => {
             console.error('Error fetching data:', error);
+            throw error; // Re-throw the error to handle it where the function is called

```

Comment:
Your catch should be at the end of your chain - you dont need to catch or do anything with the error in this file